torch/cuda/__init__.py

```diff
@@ -22,44 +22,17 @@
 from multiprocessing.util import register_after_fork as _register_after_fork
 from ._utils import _get_device_index
 
+try:
+    from torch._C import _cudart
+except ImportError:
+    pass
```

Comment:
Why is that ok to ignore? Won't the error message be quite unfriendly if something in this file tries to use `_cudart` like to check a cuda error? (same for _cudnn and _nvtx)

Reply:
It's ok to ignore, because it's not the case that it will always be there (e.g. in CPU builds). But I agree that we should be careful about error messages.